src/pages/item/index.js

```diff
@@ -12,18 +12,24 @@ import {
   Text
 } from "office-ui-fabric-react";
 
+import createPdf from "../../utils/create-pdf"
+
 import Container from "../../components/container";
 
 import FormikCheckbox from "../../components/formik/checkbox";
 import FormikTextfield from "../../components/formik/textfield";
 import FormikDatepicker from "../../components/formik/datepicker";
+import FormikDevicepicker from "../../components/formik/devicePicker";
 
 const Item = ({ itemId }) => {
   const { items, fetched } = useStoreState(store => store.macs);
   const { addMac, listMacs } = useStoreActions(store => store.macs);
 
   const item = items.find(item => item.id === itemId) || {
     owner: "",
+    CF: "",
```

Comment:
Try to use the same name convention for fields: I suggest `taxCode` or `fiscalCode`.